### ISSUE TITLE
[Feature Request][Liquid Clustering][Power Query Connector]Liquid Clustering Incompatible with Power BI

### ISSUE THREAD

**Issue by mrbenhop** [delta-io/delta#2797]
#### Liquid Clustering Connection to Power Query in Power BI
I know this may not be the most appropriate place to post but not sure where else I could get the right help. I've noticed Power BI can't read the Detla Files where liquid clustering has been applied. I'm accessing files from Blob Storage Containers in Azure Data Lake Storage Gen2. 

I know Liquid Clustering is pretty new, can't find much on this anywhere so I thought I'd post here.

**Comment by tdas** [comment]
Yeah, PowerBI needs to implement support for the protocol changes related to Liquid. Please contact PowerBI support for this. 

We understand this challenge of keeping up with protocol changes, and that's why we are building Delta Kernel project [Delta Kernel project](https://www.youtube.com/watch?v=KVUMFv7470I) to make it easier for Delta connectors to keep up with protocol changes. It would be great if PowerBI adopts Delta Kernel and then this would be easier in the future.

**Comment by raveeram-db** [comment]
@mrbenhop Please re-open if you hear back from PowerBI or are interested in building a Kernel based connector like @tdas mentions and would like any guidance.
